src/wollok/lang.wlk

```diff
@@ -241,6 +249,8 @@ class Object {
     * internal method: generates a does not understand message
     * parametersSize must be an integer value
     */
+  // TODO: Does this really need to be a native?
+  // TODO: Move to mirror?
```

Comment:
I'll consider opening a new issue for this, so we can enhance i18n in wollok-language instead of having a native solution in each implementation. I like the idea of wollok-language giving a common way to solve this.

Reply:
Nice idea